resilience4j-spring-boot2/src/test/resources/application.yaml

```diff
@@ -88,6 +88,12 @@ resilience4j.bulkhead:
       maxConcurrentCall: 1
     backendB:
       baseConfig: default
+    backendC:
+      threadPoolProperties:
```

Comment:
Not sure about this.
I think we need a new namespace like `resilience4j.threadpoolbulkhead` so that a user can also define a default configuration. 
In this way a user could also use the same name for both bulkheads. If he wants to use the backend name for both bulkhead heads, it would not be possible at the moment.

Reply:
it would be possible to use the same back-end with the 2 types of Bulkhead , i will add a test case for it as we filter on threadPoolProperties if it is defined or not .

for shared config , the user still can do the same as the normal bulkhead , just he will add  threadPoolProperties section there in the shared or default part

Reply:
I would not use `threadPoolProperties` as a sub property. I looks like the semaphore-properties `maxConcurrentCall` and `maxWaitTime` would be inherited from default, if you use `baseConfig: default`, which is not the case.
I would really separate them into two namespaces.